registrants/src/UI/embc-registrant/src/app/feature-components/needs-assessment/needs-assessment-mapping.service.ts

```diff
@@ -65,7 +65,9 @@ export class NeedsAssessmentMappingService {
             sameLastNameCheck: '',
             isPrimaryRegistrant: '',
             id: '',
-            isMinor: ''
+            isMinor: '',
```

Comment:
Isn't  `isMinor` and `isPrimaryRegistrant` Boolean ? If yes, then it is good to assign a Boolean.